doc/website/blog/2023-12-12-hibernate-schema-loader.mdx

```diff
@@ -0,0 +1,598 @@
+---
+title: "Announcing Automatic Migrations for Hibernate Users"
+authors: dorav
+tags: [hibernate, schema, migration]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info TL;DR
+
+You can now import the desired database schema from your Hibernate project into Atlas,
+and use it to automatically plan migrations for you.
+
+[See an example](#demo-time)
+
+:::
+
+## Introduction
+
+[Hibernate-ORM](https://hibernate.org/orm/documentation/), is one of the most popular ORMs for Java, so much so that parts of it have evolved into
+[the JPA standard](https://docs.jboss.org/hibernate/orm/6.4/introduction/html_single/Hibernate_Introduction.html#hibernate-and-jpa) and the jakarta APIs.
+
+Today, we are proud to announce that Atlas now supports loading and managing Hibernate schemas.
+
+[Atlas](https://atlasgo.io) is a modern tool for managing your database schema. It allows you to inspect, plan, lint and execute schema changes
+to your database. It is designed to be used by developers, DBAs and DevOps engineers alike.
+
+Atlas helps with creating standardized pipelines for managing database schemas. While it is a common practice to manage database access
+via an ORM, managing the schema changes is still a hard problem. Users often expect to be able to use the same tool to solve
+both problems, but alas, that's not always supported.
+
+By utilizing Atlas in a Hibernate project, developers can easily plan, create and analyze schema migrations changes. Finally,
+when it is time to deploy the changes, Atlas will apply the changes in the correct order, keep track of where the
+changes were applied (for example production or development databases) and allow you to visualize the current state of the database.
+
+![](https://atlasgo.io/uploads/ci-cd-guide/database-ci-cd-workflow.png)
+
+Read more about [Atlas features](/)
+
+:::note
+Hibernate support is currently in Beta and we would love to hear your feedback :heart:.
+Please reach out to us on [Discord](https://discord.gg/zZ6sWVg6NT) or by opening an [issue](https://github.com/ariga/atlas-provider-hibernate/issues/new).
+:::
+
+## Integrating Atlas into your Hibernate project
+
+Managing database schemas using Hibernate is currently not an easy task. Developers feel that [they can't trust](https://stackoverflow.com/a/221422/1708860) Hibernate to manage their schemas.
+In fact, Hibernate themselves recommend using an external tool:
+
+> Although Hibernate provides the update option for the hibernate.hbm2ddl.auto configuration property,
+this feature is not suitable for a production environment. [...] You should always use an automatic schema migration tool and have all the migration scripts stored in the Version Control System.
+>
+> — [Hibernate User Guide](https://docs.jboss.org/hibernate/orm/6.4/userguide/html_single/Hibernate_User_Guide.html)
+
+Atlas can read Hibernate schema and plan database schema migrations.
```

Comment:
This sentence feels a bit random.

I would write something along the lines of "This is where Atlas comes in. Atlas can read Hibernate..."

Reply:
Big like 👍